install.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+#
+# A basic install wrapper for UNIX-based systems.
+#
+
+GITHUB_CHANGES_FILE="https://raw.githubusercontent.com/a-scie/lift/main/CHANGES.md"
+GITHUB_DOWNLOAD_BASE="https://github.com/a-scie/lift/releases/download"
+
+# Check if curl is available.
+if ! command -v curl &> /dev/null; then
+  echo "Error: curl command not found. Please install curl and try again." >&2
+  exit 1
+fi
+
+# Check if arch is available.
+if ! command -v arch &> /dev/null; then
+  echo "Error: arch command not found. Please install arch and try again." >&2
+  exit 1
+fi
+
+# Map platform architecture to released binary architecture.
+READ_ARCH="$(arch)"
+case "$READ_ARCH" in
+  x86_64*)   ARCH="x86_64" ;;
```

Comment:
I think you want amd64 as well to cover the bases.

Reply:
rgr